gradle/libs.versions.toml

```diff
@@ -45,15 +45,15 @@ jackson = "2.18.2"
 jsonSchemaSerialization = "0.6.6"
 kaml = "0.66.0"
 keycloakTestcontainerVersion = "3.5.1"
-koin = "4.0.0"
+koin = "4.1.0-Beta1"
```

Comment:
Please mention this in the commit message and maybe also link https://github.com/InsertKoinIO/koin/issues/1716.